sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_generated/_configuration.py

```diff
@@ -26,28 +26,32 @@ class AzureSchemaRegistryConfiguration(Configuration):
     Note that all parameters used to create this instance are saved as instance
     attributes.
 
-    :param credential: Credential needed for the client to connect to Azure.
-    :type credential: ~azure.core.credentials.TokenCredential
     :param endpoint: The Schema Registry service endpoint, for example my-namespace.servicebus.windows.net.
     :type endpoint: str
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: ~azure.core.credentials.TokenCredential
+    :keyword api_version: Api Version. The default value is "2021-10". Note that overriding this default value may result in unsupported behavior.
```

Comment:
since this was added internally, is this kwarg something we should expose to the user? @yunhaoling

Reply:
does the service support multi api version now? or "2021-10" is the only supported version.

If there're multiple versions being supported already from the service side, then yes, we should do as what other http based sdks do, introducing  api version enum, document it in the docstring.

probably worthwhile to check with feature crew.

Reply:
does not support multiple api versions